DEVELOPMENT.md

```diff
@@ -1,47 +1,44 @@
 # Developing the Tekton website
 
 - [Developing the Tekton website](#developing-the-tekton-website)
-  - [Running Locally](#running-locally)
-  - [tekton.dev](#tektondev)
+- [Running Locally](#running-locally)
+- [tekton.dev](#tektondev)
 
-## Running Locally
-
-
-The [tekton.dev](#tekton.dev) is deployed on Netlify. Netlify will invoke
-[the Makefile](.Makefile) to build the website.
-
-This is configured in:
-
-* [netlify.toml](netlify.toml)
-* [runtime.txt](runtime.txt) (the python runtime)
-* [requirements.txt](requirements.txt) (the python requirements)
-
-To run it locally you will need to install:
+## Dependencies
 
-* [Hugo](https://gohugo.io/) [version 0.53](netlify.toml)
-* `npm` to install Hugo's requirements:
-  * `npm install` (in directory with [package.json](package.json))
+* `python3` [installation link](https://www.python.org/downloads/) 
+* `hugo (EXTENDED VERSION)` [installation link](https://github.com/gohugoio/hugo/releases)
+* `pip` [installation link](https://pip.pypa.io/en/stable/installing/)
+* `npm v6.14.5` [installation link](https://nodejs.org/en/)
+* `node v14.3.0` [installation link](https://nodejs.org/en/)
+* `netlify cli` [installation link](https://cli.netlify.com/getting-started)
+* `netlify account` [sign up](https://app.netlify.com/)
```

Comment:
just a nit: can we make the tool name the clickable text instead? This avoids repeating 'installation link' and is generally better for accessibility (e.g. screen readers).

Reply:
Yep, thats a good idea! :smile: